src/doc/DocAnnotator.js

```diff
@@ -148,7 +148,10 @@ class DocAnnotator extends Annotator {
 
         if (annotationType === TYPES.point) {
             let clientEvent = event;
-            if (this.hasTouch && event.targetTouches && event.targetTouches.length > 0) {
+            if (this.hasTouch && event.targetTouches) {
+                if (event.targetTouches.length <= 0) {
```

Comment:
Just a general comment about this function: we should considering chopping this up into multiple functions, it's starting to get a little unruly

Reply:
Agreed.

Reply:
Or bring in something like `getProp` from preview